src/main.rs

```diff
@@ -50,22 +86,40 @@ fn main() {
         );
     });
 
-    println!(
-        "Found .git repository in {}",
-        git_repo.repo_dir.to_str().unwrap()
-    );
+    //println!(
+    //"Found .git repository in {}",
+    //git_repo.repo_dir.to_str().unwrap()
+    //);
```

Comment:
Hmm, not sure about removing this message. I originally added this because in my use-case there are a lot of "nested" git repos, so I like to know which `.git` repository has been updated by the tool. If this is a nuisance for you, maybe we could take a look at adding some kind of configuration file, so everyone could pick the behaviour  they want?

Reply:
I just thought that since help can be triggered some other more secure ways this was unnecessary.